translations/story/04/1008/006 (my road,so long……).json

```diff
@@ -0,0 +1,497 @@
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "浜辺までひとっ走りだ！\r\nバイク……はねえから、この脚で！",
+            "enText": "Time to race to the beach! \nI… don't have a bike, so I'll just have to run!",
+            "nextBlock": 18,
+            "pathId": 5346513633398445106,
+            "blockIdx": 17
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "うおりゃああああああ！",
+            "enText": "Haaaaaaaaaah!",
+            "nextBlock": 19,
+            "choices": [
+                {
+                    "jpText": "あ、おい！",
+                    "enText": "H-hey!",
+                    "nextBlock": 19
+                },
+                {
+                    "jpText": "あ、ちょっと！",
+                    "enText": "H-hey!",
+                    "nextBlock": 19
+                }
+            ],
+            "pathId": 6845273250183935462,
+            "blockIdx": 18
+        },
+        {
+            "jpName": "",
+            "enName": "",
+            "jpText": "全速で駆け出すウオッカ。\r\nその背を必死に追いかけた。……タクシーで。",
+            "enText": "Vodka ran off at full speed, and I \ndesperately chased after her… by taxi.",
+            "nextBlock": 20,
+            "pathId": 8648559421284480148,
+            "blockIdx": 19
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "はぁ、はぁ、気持ちいいもんだな。\r\nたまにはターフじゃなくてロードを駆けるのも。",
+            "enText": "Huff, huff, that felt good. \nSometimes running on roads as opposed to turf is nice too.",
+            "nextBlock": 21,
+            "pathId": -4561517548742367865,
+            "blockIdx": 20
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "汗ばんだ肌に潮風……たまんねぇぜ。\r\nカッコいいってのは、まさにこういう……。",
+            "enText": "I just can't get enough of feeling the salty sea breeze on sweaty \nskin. This is exactly what I'm talking about when I say cool…",
+            "nextBlock": 22,
+            "pathId": -5827815542394163687,
+            "blockIdx": 21
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "…………。",
+            "enText": "…",
+            "nextBlock": 23,
+            "choices": [
+                {
+                    "jpText": "どうした？",
+                    "enText": "What's wrong?",
+                    "nextBlock": 23
+                },
+                {
+                    "jpText": "どうしたの？",
+                    "enText": "What's wrong?",
+                    "nextBlock": 23
+                }
+            ],
+            "pathId": -7850973334167763116,
+            "blockIdx": 22
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "なあ、トレーナー。ヘンなこと聞くけど……\r\n“カッコいい”と“強い”って違うのかな？",
+            "enText": "Hey, Trainer. This is a weird question, but… \nDo you think that being \"cool\" and being \"strong\" is different?",
```

Comment:
```suggestion
            "enText": "Hey, Trainer. This is a weird question, but… \nDo you think that being \"cool\" and being \"strong\" are different?",
```

Reply:
This should remain singular imo. If the sentence ended with e.g. "things" (so "... different things?") then that would've turned the sentence into plural, but right now the "cool" and "strong" parts are separated into two singular parts.